tests/parsing/errors/other/expected/spread.res.txt

```diff
@@ -49,6 +49,20 @@ Explanation: you can't collect a subset of a record's field into its own record,
 Solution: you need to pull out each field you want explicitly.
 
 
+  Syntax error!
+  tests/parsing/errors/other/spread.res:8:1-3
+
+   6 │ 
+   7 │ let myList = list{...x, ...y}
+   8 │ let list{...x, ...y} = myList
+   9 │ 
+  10 │ type t = {...a}
+
+  Lists can only have one `...` spread, and at the end.
+Explanation: lists are singly-linked list, where a node contains a value and points to the next node. `list[a, ...bc]` efficiently creates a new item and links `bc` as its next nodes. `[...bc, a]` would be expensive, as it'd need to traverse `bc` and prepend each item to `a` one by one. We therefore disallow such syntax sugar.
```

Comment:
There's a couple of syntax errors here:
* `list[a, ...bc]` doesn't exist
* `[...bc, a]` also doesn't exist